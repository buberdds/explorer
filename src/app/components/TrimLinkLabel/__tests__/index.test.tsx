import { render } from '@testing-library/react'
import { MemoryRouter } from 'react-router-dom'
import { TrimLinkLabel } from '..'

test('TrimLinkLabel snapshot', () => {
  const { container } = render(
    <MemoryRouter>
      <TrimLinkLabel label="oasis1qq2vzcvxn0js5unsch5me2xz4kr43vcasv0d5eq4" to="/home" />
    </MemoryRouter>,
  )
  expect(container).toMatchSnapshot()
})
