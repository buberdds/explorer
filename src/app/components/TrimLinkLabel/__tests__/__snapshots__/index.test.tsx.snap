// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrimLinkLabel snapshot 1`] = `
.emotion-0-MuiTypography-root-MuiLink-root {
  margin: 0;
  color: #1976d2;
  -webkit-text-decoration: underline;
  text-decoration: underline;
  text-decoration-color: rgba(25, 118, 210, 0.4);
}

.emotion-0-MuiTypography-root-MuiLink-root:hover {
  text-decoration-color: inherit;
}

<div>
  <a
    aria-label="oasis1qq2vzcvxn0js5unsch5me2xz4kr43vcasv0d5eq4"
    class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways emotion-0-MuiTypography-root-MuiLink-root"
    data-mui-internal-clone-element="true"
    href="/home"
  >
    oasis1...0d5eq4
  </a>
</div>
`;
